src/Workleap.DomainEventPropagation.Analyzers/AnalyzerReleases.Shipped.md

```diff
@@ -3,5 +3,6 @@
 ### New Rules
 
 Rule ID | Category | Severity | Notes
----------|----------|----------|-------
-WLDEP01 | Naming   | Warning  | EventDomainAttributeUsageAnalyzer, [Documentation](https://github.com/gsoft-inc/wl-domain-event-propagation)
\ No newline at end of file
+--------|----------|----------|--------------------
+WLDEP01 | Usage    | Warning  | EventDomainAttributeUsageAnalyzer, [Documentation](https://github.com/gsoft-inc/wl-domain-event-propagation)
+WLDEP02 | Usage    | Warning  | UseUniqueNameForAttributeValue, [Documentation](https://github.com/gsoft-inc/wl-domain-event-propagation)
```

Comment:
We should also add a 'Included Roslyn Analyzers' section just like in Mediatr and Mongo